src/main/webapp/js/review-loader.js

```diff
@@ -1,9 +1,14 @@
 // Fetch messages and add them to the page.
-function fetchMessages(){
+function fetchMessages(city){
   const url = '/feed';
-  fetch(url).then((response) => {
-    return response.json();
-  }).then((messages) => {
+  let header = new Headers();
+  header.append('city', city);
+
+  fetch(url, {
+    method: 'GET',
+    headers: header
+  }).then(response => response.json())
+  .then((messages) => {
     const reviewsContainer = document.getElementById('reviews-container');
     if(messages.length == 0){
      reviewsContainer.innerHTML = '<p>There are no posts yet.</p>';
```

Comment:
Nit: change "There are no posts yet" to "There are no reviews yet"

Reply:
Done